cloudbuild.yaml

```diff
@@ -19,347 +19,347 @@
-
-- name: 'gcr.io/cloud-builders/docker'
-  id: 'Wait for Servers Start'
-  entrypoint: /bin/bash
-  args:
-  - -c
-  - e2e-tests/wait_for_start.sh --HAPI_SERVER_URLS=http://sink-server-for-openmrs:8080 --OPENMRS_SERVER_URLS=http://openmrs:8080
-  waitFor: ['Launch OpenMRS Server and HAPI FHIR Sink Server for OpenMRS']
-
-- name: '${_REPOSITORY}/synthea-uploader:${_TAG}'
-  id: 'Upload to OpenMRS'
-  env:
-    - CONVERT=--convert_to_openmrs
-    - INPUT_DIR=/workspace/synthea-hiv/sample_data
-    - SINK_TYPE=OpenMRS
-    - FHIR_ENDPOINT=http://openmrs:8080/openmrs/ws/fhir2/R4
-  waitFor: ['Wait for Servers Start']
-
-- name: '${_REPOSITORY}/batch-pipeline:${_TAG}'
-  id: 'Run Batch Pipeline FHIR-search mode with OpenMRS source'
-  env:
-    - PARQUET_PATH=/workspace/e2e-tests/FHIR_SEARCH_OPENMRS
-    - OUTPUT_PARQUET_VIEW_PATH=/workspace/e2e-tests/FHIR_SEARCH_OPENMRS/VIEWS_TIMESTAMP_1
-    - SINK_PATH=http://sink-server-for-openmrs:8080/fhir
-    - SINK_USERNAME=hapi
-    - SINK_PASSWORD=hapi
-  waitFor: ['Upload to OpenMRS']
-
-- name: '${_REPOSITORY}/e2e-tests:${_TAG}'
-  id: 'Run E2E Test for FHIR-search mode with OpenMRS source'
-  env:
-    - PARQUET_SUBDIR=FHIR_SEARCH_OPENMRS
-    - FHIR_JSON_SUBDIR=FHIR_SEARCH_OPENMRS_JSON
-    - DOCKER_NETWORK=--use_docker_network
-    - SINK_SERVER=http://sink-server-for-openmrs:8080
-    - OPENMRS_TEST=--openmrs
-  waitFor: ['Run Batch Pipeline FHIR-search mode with OpenMRS source']
-
-- name: '${_REPOSITORY}/batch-pipeline:${_TAG}'
-  id: 'Run Batch Pipeline for JDBC mode with OpenMRS source'
-  env:
-    - JDBC_MODE_ENABLED=true
-    - PARQUET_PATH=/workspace/e2e-tests/JDBC_OPENMRS
-    - OUTPUT_PARQUET_VIEW_PATH=/workspace/e2e-tests/JDBC_OPENMRS/VIEWS_TIMESTAMP_1
-    - SINK_PATH=http://sink-server-for-openmrs:8080/fhir
-    - SINK_USERNAME=hapi
-    - SINK_PASSWORD=hapi
-    - FHIR_DATABASE_CONFIG_PATH=/workspace/utils/dbz_event_to_fhir_config.json
-  waitFor: ['Run E2E Test for FHIR-search mode with OpenMRS source']
-
-- name: '${_REPOSITORY}/e2e-tests:${_TAG}'
-  id: 'Run E2E Test for JDBC mode with OpenMRS source'
-  env:
-    - PARQUET_SUBDIR=JDBC_OPENMRS
-    - FHIR_JSON_SUBDIR=JDBC_OPENMRS_FHIR_JSON
-    - DOCKER_NETWORK=--use_docker_network
-    - SINK_SERVER=http://sink-server-for-openmrs:8080
-    - OPENMRS_TEST=--openmrs
-  waitFor: ['Run Batch Pipeline for JDBC mode with OpenMRS source']
-
-- name: '${_REPOSITORY}/e2e-tests:${_TAG}'
-  id: 'Test Indicators'
-  entrypoint: /bin/bash
-  args:
-  - -c
-  - 'cd dwh; ./validate_indicators.sh'
-  waitFor: ['Run E2E Test for JDBC mode with OpenMRS source', 'Turn down FHIR Sink Server Controller for e2e tests']
-
-- name: 'docker/compose'
-  id: 'Turn down Webserver and HAPI Server'
-  args: ['-f', './docker/openmrs-compose.yaml', '-f', './docker/sink-compose.yml', '-p', 'openmrs-project', 'down']
-  env:
-    - SINK_SERVER_NAME=sink-server-for-openmrs
-    - SINK_SERVER_PORT=9002
-  waitFor: ['Test Indicators']
+  - name: 'docker/compose'
```

Comment:
What are these extra indents that are added in this commit? Are these added by spotless? In the examples from [Cloud Build](https://cloud.google.com/build/docs/configuring-builds/create-basic-configuration) I don't see this convention; if these are not necessary, I prefer that we don't do it. If they are indeed something done by spotless, I prefer that we move them to a pure formatting PR submitted after this (to make it clear what is changes in this PR).